src/popup/App.vue

```diff
@@ -184,10 +194,25 @@ export default defineComponent({
 
     initVisibilityListeners();
 
+    onBeforeMount(async () => {
+      if (IS_MOBILE_APP) {
+        StatusBar.setBackgroundColor({
```

Comment:
Can it be achieved as a property somewhere in the project?

Reply:
https://ionicframework.com/docs/native/status-bar
Not that I'm aware of